docs/contributing-analyzer.md

```diff
@@ -17,28 +17,33 @@ In general, it is best to run commands from the Visual Studio Developer Command
     - ASP.NET and web development
     - .NET desktop development
     - Visual Studio extension development
-1. Ensure to install Individual components:
+1. Ensure to install *Individual components*:
     - .NET Framework 4.8 SDK
     - .NET Framework 4.8 Targeting pack
+    - .NET Framework 4.7.2 Targeting pack
+    - .NET Framework 4.6 Targeting pack
     - .NET SDK
     - .NET Compiler Platform SDK
 1. Install also:
-    - .NET Core 3.1 SDK
-    - .NET 3.5 SDK (SP1)
+    - [.NET Core 3.1 SDK](https://dotnet.microsoft.com/en-us/download/dotnet)
+    - .NET 3.5 SDK (SP1) TODO: Add link to dotNetFx35setup.exe
+    - Install Visual Studio 2019 and check these SDKs in the *individual components* tab
+        - .NET framework 4 targeting pack
+        - .NET framework 4.5 targeting pack
 1. The following environment variables must be set:
-    - **JAVA_HOME**
-    - **MSBUILD_PATH** - path to the MSBuild.exe executable (MSBuild 16)
+    - **JAVA_HOME** (e.g. `C:\Program Files\Java\jdk-11.0.2`)
+    - **MSBUILD_PATH** - path to the MSBuild.exe executable (MSBuild 16 e.g. `C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\MSBuild.exe`)
     - **NUGET_PATH** - path to the nuget.exe executable (related to the [plugin integration tests](./contributing-plugin.md#integration-tests))
-    - **ORCHESTRATOR_CONFIG_URL** - url to orchestrator.properties file (for integration tests) in uri form (i.e. file:///c:/something/orchestrator.properties)
-    - **RULE_API_PATH** - path to folder containing the rule api jar
+    - **ORCHESTRATOR_CONFIG_URL** - url to orchestrator.properties file (for integration tests) in uri form (i.e. file:///c:/something/orchestrator.properties). TODO: where to get the file, where to place it and document to put that variable in "user" instead of "system"
```

Comment:
add the link to https://github.com/sonarsource/orchestrator#configuration ? is there something else which is not documented there?

Reply:
You need a `orchestrator.artifactory.apiKey`:
```
# Token used to download SonarSource private artifacts from https://repox.jfrog.io/repox
# Generate your API key at https://repox.jfrog.io/repox/webapp/#/profile
# This property can be replaced by the environment variable ARTIFACTORY_API_KEY.
#orchestrator.artifactory.apiKey=xxx
```

I don't know if an account at jfrog is enough to get access. You propably also need access to some kind of organization/repo to make this work, but I can't tell for sure. I think we created my account via service-now and I don't know what permissions they set for my account.

Reply:
You also nee a github token:
```
# Personal access token used to request SonarSource development licenses at https://github.com/sonarsource/licenses. 
# Generate a token from https://github.com/settings/tokens
# This property can be replaced by the environment variable GITHUB_TOKEN.
github.token=xxx
```
I think that token is just fine, because the repos at github are open source (not sure if this is true for all the repos required by the ITs).

Reply:
All of this is documented with the mention maybe that:
- in `~/.sonar/orchestrator/orchestrator.properties` - `~` means `%USERPROFILE%`
- all properties need to be uncommented
- regarding repox, you should log with SAML SSO (small button below the username+password fields)

Reply:
I'll update the readme to point to https://repox.jfrog.io/ui/admin/artifactory/user_profile instead

Reply:
> regarding repox, you should log with SAML SSO

How can contributors from outside do this? The page is called contributing-analyzer.md and there is no distinction between sonarsourcers and non-sonarsourcers in the document.   How is that supposed to work for outsiders?

Reply:
As mentioned in [this comment](https://github.com/SonarSource/sonar-dotnet/pull/5563#discussion_r865619785) - it's not supposed to work for outsiders. This document is mainly used by Sonar employees.

Feel free to add a note that this works only internally.